include/deal.II/grid/tria_accessor.h

```diff
@@ -3450,13 +3450,24 @@ class CellAccessor : public TriaAccessor<dim, dim, spacedim>
    */
 
   /**
-   * Return global active cell index for an active cell.
+   * Return globally unique cell index for a non-artificial active cell. The 
+   * value is identical to active_cell_index() in the context of a serial 
+   * triangulation.
+   * 
+   * In the context of parallel triangulations, locally-owned cells are  
+   * enumerated contiguously within each subdomain of the mesh. 
+   * 
+   * If a cell-data vector has been set up with 
+   * parallel::global_active_cell_index_partitioner(), the returned index can 
```

Comment:
Do you want to add the triangulation name here?

Reply:
Yes: facepalm...

Reply:
Changed!